assets/xml/objects/object_kaizoku_obj.xml

```diff
@@ -7,13 +7,13 @@
         <Texture Name="object_kaizoku_obj_Tex_0019E0" OutName="tex_0019E0" Format="rgba16" Width="64" Height="32" Offset="0x19E0" />
         <Texture Name="object_kaizoku_obj_Tex_0029E0" OutName="tex_0029E0" Format="rgba16" Width="16" Height="32" Offset="0x29E0" />
         <Texture Name="object_kaizoku_obj_Tex_002DE0" OutName="tex_002DE0" Format="rgba16" Width="16" Height="16" Offset="0x2DE0" />
-        <DList Name="object_kaizoku_obj_DL_0032A0" Offset="0x32A0" />
-        <Collision Name="object_kaizoku_obj_Colheader_0035B0" Offset="0x35B0" />
+        <DList Name="gPirateBarredShutterDL" Offset="0x32A0" />
+        <Collision Name="gPirateBarredShutterColHeader" Offset="0x35B0" />
```

Comment:
```suggestion
        <Collision Name="gPirateBarredShutterCol" Offset="0x35B0" />
```
Collision gets suffixed by just `Col`. For a reference about xml suffixes https://github.com/hensldm/ZAPD/blob/master/docs/zapd_extraction_xml_reference.md